convokit/storage/memMappings.py

```diff
@@ -0,0 +1,94 @@
+from typing import MutableMapping, Callable
+
+
+class MemCollectionMapping(MutableMapping):
+    def __init__(self, name, item_type=None):
+        super().__init__()
+        self.name = name
+        self.item_type = item_type
+        self.data = {}
+
+    def with_storage(storage) -> Callable[[str, type], MutableMapping]:
```

Comment:
by reading through it, it seems like a static method. especially because it seems like the variables being checked in storage don't match up with "self".

Reply:
Yup, the @staticmethod decorator should be added